Test/SubscriptionTest.cs

```diff
@@ -305,6 +305,51 @@ public void UpdateSubscription()
             Assert.Equal(newSubscription.NetTermsType, NetTermsType.NET);
         }
 
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void UpdateSubscriptionWithProrationSettings()
+        {
+            var plan = new Plan(GetMockPlanCode(), GetMockPlanName())
+            {
+                Description = "Update Subscription Plan 1"
+            };
+            plan.UnitAmountInCents.Add("USD", 1500);
+            plan.Create();
+            PlansToDeactivateOnDispose.Add(plan);
+
+            var plan2 = new Plan(GetMockPlanCode(), GetMockPlanName())
+            {
+                Description = "Update Subscription Plan 2"
+            };
+            plan2.UnitAmountInCents.Add("USD", 750);
+            plan2.Create();
+            PlansToDeactivateOnDispose.Add(plan2);
+
+            var account = CreateNewAccountWithBillingInfo();
+
+            var sub = new Subscription(account, plan, "USD");
+            sub.Create();
+
+            var subChange = new SubscriptionChange()
+            {
+                PlanCode = plan2.PlanCode,
+                BillingInfoUuid = account.GetBillingInfos()[0].Id,
+                NetTerms = 10,
+                NetTermsType = NetTermsType.NET,
+                ProrationSettings = new ProrationSettings()
+                {
+                    Charge = ProrationSettings.Options.FullAmount,
+                    Credit = ProrationSettings.Options.None
+                }
+            };
+
+            Assert.Equal(subChange.ProrationSettings.Charge, ProrationSettings.Options.FullAmount);
```

Comment:
To actually test this you probably want to do the assertions last, after you get the subscription here:
`var newSubscription = Subscriptions.Get(sub.Uuid);`
Then assert with the newSubscription

` Assert.Equal(newSubscription.ProrationSettings.Charge, ProrationSettings.Options.FullAmount);`

Reply:
This is not returned on the subscription. I am only able to check these on the subscription change. We pass it in but it is not passed back in the response.